lib/helper/FileSystem.js

```diff
@@ -86,6 +94,15 @@ class FileSystem extends Helper {
     const content = getFileContents(this.file, encoding);
     fileEquals(this.file).negate(text, content);
   }
+
+  /**
+  * Returns filenames without directories. 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4515295)